consult-compile.el

```diff
@@ -67,20 +67,21 @@
 
 (defun consult-compile--error-lookup (_ candidates cand)
   "Lookup marker of CAND by accessing CANDIDATES list."
-  (when-let (cand (car (member cand candidates)))
-    (let ((marker (get-text-property 0 'consult-compile--marker cand))
-          (loc (get-text-property 0 'consult-compile--loc cand)))
-      (when (buffer-live-p (marker-buffer marker))
```

Comment:
Oh, I meant to keep the buffer live check but move the buffer-local-value inside the check.

Reply:
The nice thing is, `marker-buffer` already returns nil in that case.

Reply:
Perfect.